docs/access_control/Http_Exchanges.md

```diff
@@ -0,0 +1,106 @@
+# Intro
+
+The goal of this document is to specify and represent the HTTP requests in the context of the access control.
+
+In this initial phase the token issuing endpoint is not used as the client applications have the tokens statically coded.
+
+Also, the HTTP Exchanges between the Integration team and the Core backend with the Write API using the access token are unsafe as there is no protection in the transport and network layers (TLS/SSL), the write token can be eavesdropped.
```

Comment:
Both Read and Write APIs will be exposed using HTTPS

Reply:
Indeed, forgot to update it since the last meeting on GCP.